src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -253,27 +268,20 @@ def __call__(
 
         pixel_values = self.image_processor(
             images,
-            do_resize=do_resize,
-            do_normalize=do_normalize,
-            return_tensors=return_tensors,
-            image_mean=image_mean,
-            image_std=image_std,
-            input_data_format=input_data_format,
-            data_format=data_format,
-            resample=resample,
-            do_convert_rgb=do_convert_rgb,
+            **kwargs["image_kwargs"],
         )["pixel_values"]
-
+        max_length = output_kwargs.get("max_length", None)
         if max_length is not None:
             max_length += self.image_seq_length  # max_length has to account for the image tokens
-
+        output_kwargs = self._merge_kwargs(
+            PaliGemmaProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
+            **kwargs,
+        )
```

Comment:
```suggestion
        if output_kwargs["text_kwargs"].get("max_length", None) is not None:
            output_kwargs["text_kwargs"]["max_length"] += self.image_seq_length
```
Instead of merging the kwargs again